Gruntfile.js

```diff
@@ -78,6 +79,17 @@ module.exports = function (grunt) {
     return requireOptions;
   }
 
+  var WIDGET_VERSION_TEXT = '<%= widgetversion %>';
+
+  function escapeRegexText(text) {
+    return text.replace(/([.*+?^=!:${}()|\[\]\/\\])/g, '\\$1');
+  }
+
+  function substituteWidgetVersion(content) {
+    var widgetVersionRegex = new RegExp(escapeRegexText(WIDGET_VERSION_TEXT));
```

Comment:
Without a regex, `replace` only replaces the first instance. In this case, I guess a simple replace would work, because we only expect a single instance of the widget version.

Will change